packages/odyssey-react-mui/src/labs/DataFilters.tsx

```diff
@@ -390,6 +428,8 @@ const DataFilters = ({
               {/* Filter popover */}
               <MuiPopover
                 anchorEl={filterPopoverAnchorElement}
+                elevation={2}
+                sx={{ marginLeft: 2, marginTop: -1 }}
```

Comment:
A bit on the magic side. I think I know why this is needed but, maybe worth a small comment about what this is doing would be helpful